### PR TITLE
Review

### PR DESCRIPTION
**Pull request by mamidishettikarthik**
Home assignment using HTML and CSS.

Screenshot (87)

Screenshot (88)

Screenshot (89)

Screenshot (90)

Screenshot (84)

Screenshot (91)

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    height: 46px;
+    background-color: white;
+    border: 1.5px solid red;
+}
+#bluebox {
+    margin: 0%;
+    padding: 0%;
+    font-size: 20px;
+}
+#graybox {
```
Comment (ghost) [inline CSS/style.css]: Try to use proper naming convention. Here graybox and bluebox does depicts section it represents

File: CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    font-size: 10px;
+    font-weight: bold;
+}
+.container{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr
+}
+.Container_One{
```
Comment (ghost) [inline CSS/style.css]: Follow the consistent naming conventions

**Review by ghost — COMMENTED** [review]
you can create a separate sections to build a main page. For now its fine

File: CSS/style.css (hunk 1 above)
Comment (mamidishettikarthik) [reply CSS/style.css]: Naming conversations are changed.
Screenshot 2023-02-14 105753

File: CSS/style.css (hunk 2 above)
Comment (mamidishettikarthik) [reply CSS/style.css]: Screenshot 2023-02-14 110123

File: CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+}
+#contentMiddle{
+    font-size: 35px;
+    display: grid;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    text-align: center;
+    position:relative;
+    margin-bottom: 4%;
+}
+.contentRight-text{
```
Comment (ghost) [inline CSS/style.css]: contentRight-text 
use either camel caps : Example: contentRightText 
or use -separated words: Example: content-right-text

**Pull request merged**